pkg/pillar/types/downloadertypes.go

```diff
@@ -219,4 +219,5 @@ type DatastoreContext struct {
 	APIKey          string
 	Password        string
 	Region          string
+	IsLocal         bool
```

Comment:
A comment might make sense to indicate that this means the datastore it hosted on some network instance on EVE itself.

Reply:
Sure.